client/flutter/lib/pages/onboarding/notifications_page.dart

```diff
@@ -54,8 +54,8 @@ class _NotificationsPageState extends State<NotificationsPage> {
   }
 
   void _registerFCMToken() async {
```

Comment:
This isn't a blocker -- but it would be nice if we could just refactor `updateFCMToken` into a separate service and just call that method over here instead of `_registerFCMToken`.

Reply:
Yep. Didn't want to do a larger code change at this point. Agree should be done. Will create an "issue".